tests/Integrations/CLI/Symfony/V4_4/CommonScenariosTest.php

```diff
@@ -28,33 +29,42 @@ public function testCommand()
                     'console',
                     'cli',
                     'console'
-                )->withChildren([
+                )->withExactTags([
+                    Tag::COMPONENT => 'laravel'
```

Comment:
This is looks wrong - Laravel in a Symfony test?

Reply:
Fixed this. Due to laravel autoloading and setting the component tag in the autoloading portion and not trace/hook_method